lib/pay/lemon_squeezy/subscription.rb

```diff
@@ -0,0 +1,181 @@
+module Pay
+  module LemonSqueezy
+    class Subscription
+      attr_reader :pay_subscription
+
+      delegate :active?,
+        :canceled?,
+        :on_grace_period?,
+        :on_trial?,
+        :ends_at,
+        :name,
+        :owner,
+        :pause_starts_at,
+        :pause_starts_at?,
+        :processor_id,
+        :processor_plan,
+        :processor_subscription,
+        :prorate,
+        :prorate?,
+        :quantity,
+        :quantity?,
+        :trial_ends_at,
+        to: :pay_subscription
+
+      def self.sync_from_transaction(transaction_id)
+        transaction = ::Paddle::Transaction.retrieve(id: transaction_id)
```

Comment:
I suppose this method was copy-pasted from `paddle`, not quite sure that it should be here :)

Reply:
Yep it was, have removed this now